ui/admin/app/styles/app.scss

```diff
@@ -630,11 +630,15 @@
     min-height: 4rem; // 1
   }
 
+  // these styles are not applied directly to the order-series-diagram
+  // because these are unique to the target form and may not be
+  // needed in other instances of the diagram
   .worker-wrapper {
-    display: flex;
-    align-items: center;
-    min-height: 11rem;
-    margin-top: sizing.rems(xs) - sizing.rems(xxxxs); // 10
+    .ordered-series-diagram {
+      align-items: center;
+      min-height: 11rem;
+      margin-top: sizing.rems(xs) - sizing.rems(xxxxs); // 10
```

Comment:
Can we use rems directly when applied to HDS?  The diagrams are based on HDS, but the sizing helper is based on Structure.

Reply:
Moved style changes out of this PR.